### ISSUE TITLE
Right-Alt doesn't work for adding multiple cursors (necessary for ChromeOS)

### ISSUE THREAD

**Issue by Zacqary** [c9/core#98]
Left-Alt+Click on ChromeOS maps to a right click. This makes it impossible to add multiple cursors on Cloud9. This behavior is built into ChromeOS and is impossible to disable.

Right-Alt+Click doesn't map to a right click on ChromeOS, but it doesn't add a second cursor in Cloud9. Allowing users to add a second user using Right-Alt instead of just Left-Alt should be an effective workaround when using Cloud9 on a Chromebook.

**Comment by nightwing** [comment]
Ace accepts both left and right Alt as click modifiers. It just uses `event.altKey` property [here](https://github.com/c9/core/blob/02d1ba40/node_modules/ace/lib/ace/mouse/multi_select_handler.js#L43).
Which keyboard layout do you use? with some keyboard layouts left alt is interpreted as alt-gr and sets both `event.altKey` and `event.ctrlKey` to true.

**Comment by Zacqary** [comment]
You're absolutely right, I had US Extended on. I feel really stupid.

That probably means ChromeOS actually does interpret Right Alt as a right
click as well, so that's back to square one.

Is there a way to change the key binding for clicking to add multiple
cursors? I wasn't able to find it in the Preferences pane.

**Comment by danferth** [comment]
Yup this one is driving me crazy too. Maybe custom keybindings in c9?

**Comment by danferth** [comment]
yes you can! open preferences and select key bindings to edit `addCursorBelow` and whatnot, I changed it to `ctrl+alt+l` and boom **bobs your uncle!**
